hosting_docs/source/support/3-components-of-support-system.rst

```diff
@@ -0,0 +1,162 @@
+
+The priority level helps the entire support team and developers to understand how they should prioritize the particular ticket. A support team member triaging the ticket can setup the priority.
+
+Ticket Workflow
+````````````````
+Once the support system is set up below is a general process that can be followed. Note that for P1/P2 we recommend a separate on-call like process stated in the `P1/P2 Process`_ recommendations.
+
+.. image:: ./images/local_hosting_support_workflow.png
+   :target: ./images/local_hosting_support_workflow.png
+
+* An issue is reported view UI or directly
+* A ticket is created in helpdesk app automatically or support creates it if the issue is reported via email/chat.
+* When a new ticket arrives,
+
+  * A support team member performs the initial investigation
+  * If more information is required to resolve the issue the user is contacted for more information.
+  * If the ticket fits P1/P2 criteria, follow P1/P2 process
+  * Support team member updates the fields of the ticket such as priority, status and assignee.
+  * Depending on the ticket, the support team member might resolve and respond back to the user or escalate it to a different team member from the program or developer team.
+  * If the ticket priority is low, the team might put it into a backlog that can be reviewed later.
+  * If the team is not able to get resolve, the ticket can be reported to Dimagi support directly if the team has a support plan or else to the public CommCare developers forum
+
+* Once the resolution is found the support team member sends the resolution to the user and closes the ticket after updating relevant ticket fields.
+
+Apart from this a regular periodical (weekly or biweekly) team calls could also be used to coordinate the overall support activities.
+
+P1/P2 Process
+`````````````
+
+The standard support process stated above works well for tickets with priority lower than P2. As defined above tickets with priority P1 indicate a very urgent ticket that affects all users, which may be causing a downtime or irreversible data loss/corruption or other critical issues. P2 priority indicates a critical function being available that might soon result in a P1 issue if neglected. Given that there is a lot of urgency tied to P1 and P2, we recommend a separate process to resolve these issues.
+
+The intention of a separate P1/P2 process is to address below unique expectations associated with  P1 or P2 incidents.
+
+1. Fix the issue as soon as possible
+2. Establish communication with users and stakeholders to inform about the issue
+3. Followup Actions such as Root Cause Analysis to prevent issues like this from getting repeated
+
+We recommend below a general process that addresses these three expectations. You may tweak it as you see fit in your organizational context or even create your own process but in the least it should address the above three expectations.
+
+Process for P1/P2
+##################
+
+1. Kickoff the process
```

Comment:
```suggestion
1. Kick off the process
```
(The verb is "kick". It becomes clearer when you think about other forms of the verb. e.g. The past tense would be "kicked off the process", not "kickoffed the process", or the present participle would be "kicking off", not "kickoffing".)